lib/Form/RadioButton/Other.js

```diff
@@ -22,6 +22,29 @@ class Other extends Component {
   render() {
     const { name, value, id, checked, label, onChangeHandler, onTextChangeHandler } = this.props;
```

Comment:
I think in the future (and maybe for this PR) we begin spreading certain attributes/props rather than explicitly adding them to the component. This means any future attributes we add will always be used on the input e.g. `disabled` right now would require code changes for this to be reflected in the browser.

How would this look?

```
const { label, onChangeHandler, onTextChangeHandler, ...rest } = this.props; // rest = { name, value, id, checked }

<Input
  ...rest
  onChangeHandler={onChangeHandler}
/>
<input
  ...this one doesn't share the same props so set this as you are
/>
```
What do you think?